documentation/misc/gas_balancing.md

```diff
@@ -0,0 +1,54 @@
+## Gas Balancing
+
+The gas balancing process targets to set gas costs of syscalls to be in line with
+10 gas per nanosecond on reference hardware.
+The process can be either performed for all syscalls based on existing messages and chain or targeted
+at single syscall.
+
+#### Reference hardware
+
+The reference hardware is TR3970x with 128GB of RAM. This is what was available at the time and
```

Comment:
What do we mean by "the time"?  Can we give a date (e.g., 202007 on the rampup to mainnet launch ← this is just me making something up)

Reply:
both at the physical time (~06-2020) and what we had in our hands